src/components/ladder/CountryRankingsGrid.vue

```diff
@@ -0,0 +1,535 @@
+<template>
+  <div class="custom-table-wrapper elevation-1">
+    <table
```

Comment:
Is there no way to use RankingsGrid component here instead of copying the whole logic?
https://github.com/w3champions/w3champions-ui/blob/master/src/components/ladder/RankingsGrid.vue